src/Components/Footer/styles.js

```diff
@@ -0,0 +1,58 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    backgroundColor: '#101820',
+    width: '100%',
```

Comment:
add `flexWrap: 'wrap',` and footer should become responsive other then that we use 80% for content.

Reply:
@annabauza The measurements that were given in the issue sum up to around 75% (of wide laptop screen). Should I go with that or change the width explicitly to 80%? Also, the empty space on the left and right should be covered in the footer color or just plain white?

Reply:
@tichnas Add flexWrap: 'wrap'. Yes change the width to 80% now. I think footer color is what you should go for. Please push these changes.